README.md

```diff
@@ -53,9 +53,11 @@ Because some dependencies have broken the build in minor/patch releases, to
 compile with 1.29.0 you will need to run the following version-pinning command:
 ```
 cargo update --package "cc" --precise "1.0.41"
+cargo update --package "log:0.4.x" --precise "0.4.13" # x being the highest patch version
 cargo update --package "cfg-if" --precise "0.1.9"
 cargo update --package "unicode-normalization" --precise "0.1.9"
 cargo update --package "serde_json" --precise "1.0.39"
 cargo update --package "serde" --precise "1.0.98"
 cargo update --package "serde_derive" --precise "1.0.98"
+cargo update --package "byteorder" --precise "1.3.4"
```

Comment:
I just fixed the missing documentation of behavior already present in tests in the same commit. I can split it off if you want (just noticed after opening the PR).